Formula/gnupg.rb

```diff
@@ -38,12 +38,12 @@ class Gnupg < Formula
     depends_on "libidn"
   end
 
-  # Fixes a build failure without ldap.
+  # Fixes in-source-tree builds
   # Committed upstream, will be in the next release.
-  # https://dev.gnupg.org/T6239
+  # https://dev.gnupg.org/T6313
   patch do
-    url "https://dev.gnupg.org/rG7011286ce6e1fb56c2989fdafbd11b931c489faa?diff=1"
-    sha256 "407011d4ae9799f50008b431df60cd5b781dca0f572e956fd46245aa209af7e8"
+    url "https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=patch;h=e89d57a2cb10bd04d266165015f159be2ab48984"
+    sha256 "3316f59ba64e698187832471585ecfcb1756460954fcc69b336e1e9860872713"
   end
 
   def install
```

Comment:
Please replace `"--disable-dependency-tracking"` and `--prefix` in `./configure` with `*std_configure_args`

Reply:
Replaced.